Registrar.Database/Stores/RegiSqlUserStore.cs

```diff
@@ -16,19 +18,18 @@ public RegiSqlUserStore(string connectionString) : base(connectionString)
         {
         }
 
-        public async Task<RegiUser> GetUserById(int id)
+        private async Task<IEnumerable<RegiUser>> GetUserByQuery(string query, object parameters)
         {
             try
             {
                 using (var connection = await GetConnectionAsync())
                 {
-                    var result = await connection.QueryAsync(RegistrarQueries.UserGetById, new {Id = id});
+                    var result = await connection.QueryAsync(query, parameters);
 
                     if (!result.Any())
                         throw new RecordNotFoundException();
```

Comment:
Probably should throw exception in public methods, in case one isn't wanted/needed. For example if selecting a list of all users, probably don't want an exception, just an empty list.